[TW-794] cycle 8 edits (#4454)
postmanlabs/postman-docs: src/pages/docs/writing-scripts/pre-request-scripts.md

## Next steps

> After learning the basics of writing pre-request test scripts, you may be interested in extending your scripts:

Reviewer: Maybe this or something similar?

```suggestion
After learning the basics of writing pre-request test scripts, you could extend your scripts:
```

It keeps it kind of in line with the other phrasing you revised elsewhere (such as in the `src/pages/docs/writing-scripts/test-scripts.md` doc).

Author: I'm leaning toward "After &lt;doing the thing&gt;, **you can** &lt;do the next thing&gt;..." There was [some discussion](https://postman.slack.com/archives/C01R7E8QLF7/p1670510772436699) about this in #team-konrath today.
